src/linear_algebra/matrix/adjugate/basic.lean

```diff
@@ -4,7 +4,6 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Anne Baanen
 -/
 import algebra.regular.basic
-import linear_algebra.matrix.mv_polynomial
 import linear_algebra.matrix.polynomial
```

Comment:
Since we already rely on `polynomial` here, I don't think there's much value to cutting the dependency on `mv_polynomial`.